aerogear/aerogear-unifiedpush-server
# Adding dev image for Keycloak and UPS

* building an image, from WF_11, and installing Keycloak overlay distribution.
* deploying UPS war file

How to test:

* Run the build
```
 mvn clean install -DskipTests && docker run -p 8080:8080 -it aerogear/ups:kc
```
* Login (admin:123) and change the password
* create an app
* see this in the console:
```
10:46:28,660 DEBUG [org.jboss.aerogear.unifiedpush.service.impl.SearchManager] (default task-46) Running with Keycloak context
```
